### ISSUE TITLE
Prototypical Networks?

### ISSUE THREAD

**Issue by WangTianduo** [cyvius96/few-shot-meta-baseline#4]
Hi,

Thanks for your work.

Recently I am reading the paper "Prototypical Networks". I found your idea is quite similar to that one. Could you help explain a little bit about the main difference between your idea and that one?

What will the accuracy of ProtoNets on miniImageNet be if the backbone function is changed to ResNet that used in your paper?

**Comment by cyvius96** [comment]
You can check Appendix B for details.

For the results, the original implementation of ProtoNet on miniImageNet is about 53.77 / 75.71 for 1-shot / 5-shot. Table 6 also demonstrates a comparison to a "improved ProtoNet" (with classification pre-training and introducing the temperature on logits).

**Comment by WangTianduo** [comment]
Thanks for your prompt response. That helps a lot!

**Comment by cyvius96** [comment]
You're welcome.
